docs/pipenv_readme.md

```diff
@@ -0,0 +1,53 @@
+# Presidio analyzer - Pipenv installation
+[Pipenv](https://pipenv.readthedocs.io/en/latest/) is a Python workflow manager, handling dependencies and environment for python packages. This tutorial describes how to install presidio-analyzer locally for development purposes.
+### 1. Install pipenv
+#### Using Pip:
+```
+$ pip install --user pipenv
+```
+#### Homebrew
+```
+$ brew install pipenv
+```
+
+Additional installation instructions: https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv
+
+
+### 2. Create virtualenv for the project
+From the python project's root folder, run:
+```
+$ pipenv shell
```

Comment:
Also explain how to deactivate the environment.

Reply:
fixed (removed the shell command, so no need for deactivation)

Reply:
```suggestion
$ pipenv shell
```

To Deactivate your virtual environment, run:
```
deactivate
```